app/views/unattended/provisioning_templates/registration/global_registration.erb

```diff
@@ -41,6 +42,7 @@ register_host() {
        <%= "--data \"host[location_id]=#{@location.id}\"" if @location -%>
        <%= "--data \"host[hostgroup_id]=#{@hostgroup.id}\"" if @hostgroup -%>
        <%= "--data \"host[operatingsystem_id]=#{@operatingsystem.id}\"" if @operatingsystem -%>
+       <%= "--data \"remote_execution_interface=#{@remote_execution_interface}\"" if @remote_execution_interface -%>
```

Comment:
I'd use `@remote_execution_interface.present?`, same below, otherwise the data is added always, since the text field is empty string, which is truthy